pkg/smartcontract/manifest/standard/check.go

```diff
@@ -0,0 +1,15 @@
+package standard
+
+import "github.com/nspcc-dev/neo-go/pkg/smartcontract/manifest"
+
+// Standard represents smart-contract standard.
+type Standard struct {
+	// Manifest describes mandatory methods and events.
+	manifest.Manifest
+	// Base contains base standard.
+	Base *Standard
```

Comment:
Do we really need this field?

Reply:
This is for method/events reuse. `append`s look ugly.